tests/pipelines/test_integration.py

```diff
@@ -0,0 +1,106 @@
+        'question': 'Which NFL team represented the AFC at Super Bowl 50?',
+        'id': '0'},
+        {'context': 'Architecturally, the school has a Catholic character. Atop the Main Building\'s gold dome is a golden statue of the Virgin Mary. Immediately in front of the Main Building and facing it, is a copper statue of Christ with arms upraised with the legend \"Venite Ad Me Omnes\". Next to the Main Building is the Basilica of the Sacred Heart. Immediately behind the basilica is the Grotto, a Marian place of prayer and reflection. It is a replica of the grotto at Lourdes, France where the Virgin Mary reputedly appeared to Saint Bernadette Soubirous in 1858. At the end of the main drive (and in a direct line that connects through 3 statues and the Gold Dome), is a simple, modern stone statue of Mary.',
+        'question': 'To whom did the Virgin Mary allegedly appear in 1858 in Lourdes France?',
+        'id': '1'}
+    ]
+
+@pytest.fixture(scope="module")
+def integration_expected_single_inference():
+    return {
+        'answer': SpanTuple(text='would have been known as', start=667),
+        'score': 0.00032179668778553605
+    }
+
+@pytest.fixture(scope="module")
+def integration_expected_multi_inference():
+    return [
+        {'answer': SpanTuple(text='would have been known as', start=667),
+        'score': 0.00032179668778553605},
+        {'answer': SpanTuple(text='end of the main drive (and in a direct line that connects', start=558),
+        'score': 0.0003353202191647142}
+    ]
+
+def test_integration(
+        tmp_path,
+        integration_training_result,
+        integration_single_inference_input,
+        integration_expected_single_inference,
+        integration_multi_inference_input,
+        integration_expected_multi_inference
+    ):
+    set_seed(100)
+    experiment_dir = tmp_path
+    config_path = FIXTURES_ROOT / "pipelines/experiment.jsonnet"
+
+    checkpoint_dir = os.path.join(experiment_dir, "checkpoints")
+    output_dir = os.path.join(experiment_dir, "outputs")
+
+    ext_vars = {
+        # Used to feed the jsonnet config file with file paths
+        "OUTPUT_PATH": output_dir,
+        "CHECKPOINT_PATH": checkpoint_dir
+    }
+
+    result = run_experiment(
```

Comment:
This line crashes with `FileNotFoundError: Couldn't find a dataset script at trapper/tests/test_fixtures/hf_datasets/squad_qa_test_fixture/squad_qa_test_fixture.py or any data file in the same directory.
`

Reply:
This looks like the test is not running in the project root as the active directory. Due to experiment config specifying a relative path, that might be the cause of that problem. I specified the path relative to the project root due to where we are running the test in GitHub test action.